client/src/styles/components/chat-bubble/chat-bubble.module.scss

```
@@ -0,0 +1,34 @@
+@import '@/styles/theme.scss';
+
+.chat-bubble {
+  &.me {
+    .chat-text-wrapper {
+      border-radius: 8px 0px 8px 8px;
+      background-color: $color-primary1;
+    }
+    .chat-text {
+      color: $color-white;
+    }
+  }
+
+  &.other {
+    .chat-text-wrapper {
+      border-radius: 0px 8px 8px 8px;
+      background-color: $color-white;
+    }
+    .chat-text {
+      color: $color-title-active;
+    }
+  }
+}
+
+.chat-text-wrapper {
+  display: inline-block;
+  padding: 6px 7px 8px 7px;
+  border: 1px solid $color-primary1;
+}
+
+.chat-text {
+  font-size: 14px;
+  font-weight: 500;
+}
```

```suggestion

  overflow-wrap: break-word;
  word-break: break-word;
}
```

띄어쓰기 없이 긴 글자 왔을때 화면을 뚫고 나가는 것을 방지해야 합니다.